includes/class-wc-beta-tester-admin-menus.php

```diff
@@ -152,16 +152,20 @@ private function get_plugins_ssr_info( $active_plugins, $untested_plugins ) {
 	 * @return string
 	 */
 	protected function construct_ssr() {
+		if ( version_compare( WC()->version, '3.6', '<' ) ) {
+			return '';
+		}
+
 		$transient_name = 'wc-beta-tester-ssr';
 		$ssr            = get_transient( $transient_name );
 
 		if ( false === $ssr ) {
 			// When running WC 3.6 or greater it is necessary to manually load the REST API classes.
-			if ( version_compare( WC()->version, '3.6', '>=' ) && ! did_action( 'rest_api_init' ) ) {
+			if ( ! did_action( 'rest_api_init' ) ) {
 				WC()->api->rest_api_includes();
 			}
 
-			if ( ! class_exists( 'WC_REST_System_Status_Controller', false ) ) {
+			if ( ! class_exists( 'WC_REST_System_Status_Controller' ) ) {
```

Comment:
I think we should keep the autoload off.

Reply:
@claudiosanches It didn't work for me in master with it off. Can you try it with the `false`?